### ISSUE TITLE
Return To Home

### ISSUE THREAD

**Issue by MattPiscopo** [smealum/ninjhax2.x#6]
Après être retourné au menu home via la fonction du 1.1.0 (*hax 2.6) et ensuite relancé un jeu ma N3DSXL (v.10-6-0-31E) est restée sur un écran noir.Aucun moyen de la débloquer sauf en forçant le reboot.

**Comment by MattPiscopo** [comment]
EDIT : Pour l'écran noir,il faut avoir pris un screenshot via le HomeBrew Menu (bouton home donc),le quitter avec Start + X et boot un jeu.

**Comment by fincs** [comment]
Peut-être APT encore une autre fois devient pété...
Next time please post bug reports in English, otherwise most people can't understand them.

**Comment by MattPiscopo** [comment]
English : After going back to the home menu with the new feature from HB 1.1.0 (*hax 2.6) and then launched a game on my N3DSXL (10-6-0-31E) it got stuck on a black screen.Only way to fix it is to hard reboot the console

EDIT : For the black screen,you need to have taken a screenshot with the home button (Homebrew Channel) then exit the HBC with Start + X and boot a game.

**Comment by gnmmarechal** [comment]
Can confirm. Black screen for me as well. Whenever I exit the HBL I get a black screen after booting an app or puting my DS to sleep.
2DS 10.3.0-31 Frankenfirm. (Updated NVer, Mint, eShop and friends module)

**Comment by yellows8** [comment]
@gnmmarechal Even without pressing the HOME button while running \*hax payload?(What are you using to boot into the \*hax payload?)

**Comment by gnmmarechal** [comment]
@yellows8 Yep. I use both Menuhax and Ironhax. I'm not sure which of them is giving me the problem, I use both a lot. I'll try to find that out.

**Comment by gnmmarechal** [comment]
@yellows8 seems to be quite random, with both Ironhax and Menuhax. Both use 2.6. Can't seem to reproduce the error consistently.

**Comment by yellows8** [comment]
Did it ever occur when you never pressed the HOME button under \*hax payload?(I've been using this ret2menu for other things and it's been working fine)

**Comment by gnmmarechal** [comment]
@yellows8 I never press HOME.
